os/include/sys/socket.h

```diff
@@ -243,6 +279,9 @@ int getsockopt(int sockfd, int level, int option, FAR void *value, FAR socklen_t
 /**
 * @brief  get the socket name
 *
+* @details @b #include <sys/socket.h>\n
+* SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised